meeting-notes/2019-04-01--libp2p-sync.md

```diff
@@ -0,0 +1,116 @@
+# 2019-04-01 libp2p Sync Call
+
+[Issue for this call](https://github.com/libp2p/team-mgmt/issues/16)
+
+**Moderator**: @mgoelzer
+
+**Notetaker**:  @jhiesey
+
+**Recording**:  https://youtu.be/SdUN6N0Dc8o
+
+**Attendees**:
+  - @jhiesey
+  - @mgoelzer
+  - @anacrolix
+
+## Agenda
+
+- Weekly updates 
+
+## Weekly updates
+
+- @marten-seemann:
+  - Done:
+    * attented the IETF meeting in Prague
+    * quic-go has interop with most other QUIC implementations
+  - Blocked:
+    * I will merge https://github.com/libp2p/specs/pull/151 very soon, if there aren't any futher objections
+  - Next:
+    * use quic-trace to improve quic-go performance
+
+- @yusefnapora:
+  - Done:
+    - merged in a couple concept docs
+      - addressing: https://docs.libp2p.io/concepts/addressing/
+      - peer id: https://docs.libp2p.io/concepts/peer-id/
+  - Next:
+    - finish stream muxing concept doc PR: https://github.com/libp2p/docs/pull/40#event-2244416671
+    - start on protocol negotiation & encryption concept docs
+
+- @anacrolix:
+  - Done:
+    - Misc bug fixes
+    - Exploring metrics in Prometheus and OC
+  - Blocked:
+    - Meaningful results from changes to streams
+  - Next:
+    - Merge in metrics to DHT
+    - Explore tracing?
+    - Expose metrics in related repos to DHT
+    - https://github.com/libp2p/go-libp2p-kad-dht/issues/295
+    - https://github.com/libp2p/go-libp2p-kad-dht/issues/283
```

Comment:
@anacrolix this is going to be complex to implement with the current system while avoiding overhead. One possibility is to add support for TCP session resumption, thus bypassing SecIO and multiplexer negotiation on reconnect. We'll need an abstract API for this that QUIC can also use.

cc @jhiesey @marten-seemann